script/get-platform-details/_cm.json

```diff
@@ -4,17 +4,44 @@
   "automation_uid": "5b4e0237da074764",
   "cache": false,
   "category": "Platform information",
+  "input_mapping": {
+    "out_dir_path": "CM_PLATFORM_DETAILS_DIR_PATH",
+    "out_file_name": "CM_PLATFORM_DETAILS_FILE_NAME"
+  },
   "deps": [
     {
       "tags": "detect,os"
     },
     {
+      "skip_if_env": {
```

Comment:
We should not omit the dependency in this case. Instead, we can introduce run-macos and run-windows scripts within the get-platform-details module. Initially, these scripts can start as empty files. Over time, we can incrementally add commands that are specific to macOS and Windows to these scripts.